scripts/demonstrations/stub_substitution.py

```
@@ -0,0 +1,168 @@
+    pipeline_b = Pipeline(
+        layers=[PatternLayer(), EntityLayer(), Article9Layer(provider), StubCombinationLayer()],
+        aggregator=Aggregator(),
+    )
+
+    print(f"\nKörning A: CombinationLayer ({_MODEL})...")
+    results_a = []
+    for i, text in enumerate(texts, 1):
+        print(f"  [{i:2d}/{len(texts)}] {truncate(text, 60)}")
+        results_a.append(classify_text(pipeline_a, text))
+
+    print(f"\nKörning B: StubCombinationLayer...")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)